src/trunk.jl

```diff
@@ -129,6 +129,20 @@ end
   return solve!(solver, nlp; x = x, kwargs...)
 end
 
+"""
```

Comment:
Could you move this function here https://github.com/JuliaSmoothOptimizers/JSOSolvers.jl/blob/main/src/JSOSolvers.jl ? I think it could be beneficial for all the solvers.